### ISSUE TITLE
Model has to be selected every time interpreter is started

### ISSUE THREAD

**Issue by kjyv** [KillianLucas/open-interpreter#337]
### Describe the bug

Interpreter always asks to select a model when starting `interpreter --local` (or when setting `INTERPRETER_CLI_LOCAL_RUN=True` in .env - IMO it would be nicer to have a proper config file e.g. `.open-interpreter` instead of `.env`). There is no .env parameter to select the model. Even when e.g. running `interpreter --model TheBloke/CodeLlama-13B-Instruct-GGUF`, it still asks again which model I want to load, if I want to use GPU etc.

### Expected behavior

Ideally, it does not ask anything once the model has been chosen once, everything should be written to a config file automatically.

### Open Interpreter version

0.1.3

### Python version

3.11.3

### Operating System name and version

macOS 13.5.2

**Comment by guoqiao** [comment]
I also noticed this issue.

tldr: I tried out, if you specify the model file name, then it will stop asking. E.g.:
```
interpreter --model codellama-13b-instruct.Q2_K.gguf                    
Model set to CODELLAMA-13B-INSTRUCT.Q2_K.GGUF
```
To show all models you already downloaded, for example, on linux:
```
ls -1 ~/.local/share/'Open Interpreter'/models/
codellama-13b-instruct.Q2_K.gguf
codellama-13b-instruct.Q4_K_M.gguf
codellama-13b.Q2_K.gguf
codellama-13b.Q8_0.gguf
codellama-34b-instruct.Q4_K_M.gguf
codellama-34b-instruct.Q8_0.gguf
codellama-34b.Q2_K.ggufruwh72vy.tmp
codellama-7b-instruct.Q2_K.gguf
codellama-7b-instruct.Q4_K_M.gguf
codellama-7b.Q2_K.gguf
codellama-7b.Q2_K.gguf46l6ajz8.tmp
falcon-40b-Q2_K.gguf
```

However,  the `--model` option is indeed confusing to use in a few ways:

1. in `interpreter --help`,  it's unclear what it should be:
`--model MODEL         model name (for OpenAI compatible APIs) or HuggingFace repo`

2.  When following the cli prompt, it prints name like `TheBloke/CodeLlama-13B-Instruct-GGUF`.  But when you provide the value to `--model`, it doesn't work. And it's unclear what the other options are.

3.  We see multiple ways to present a "model":
      a. lowercase file name: codellama-13b-instruct.Q2_K.gguf
      b. upper case file name: CODELLAMA-13B-INSTRUCT.Q2_K.GGUF
      c. repo or builtin metadata name (?):  TheBloke/CodeLlama-13B-Instruct-GGUF


Some thoughts to improve the situation:
- uniform the way to specify/present a model
- provide a cli option to list all available models
- just use `--model` only to specify a model, no need to have `--fast`, `--local`, `--falcon`, etc.

For example, to list models:
```
interpreter --list-models

gpt-3.5-turbo
gpt-4
codellama-13b-instruct.Q2_K.gguf
falcon-40b-Q2_K.gguf
...
```
(here I just list names as example, maybe it can provide some extra info, like size, url, mem requirement, etc.)

To use different models:
```
interpreter --model gpt-3.5-turbo
interpreter --model gpt-4
interpreter --model codellama-13b-instruct.Q2_K.gguf
interpreter --model falcon-40b-Q2_K.gguf
```

**Comment by kjyv** [comment]
Sorry, this doesn't work, at least on macOS with the current release (now 0.1.4).
I still see this:
```
$ ls -1 ~/Library/'Application Support'/'Open Interpreter'/models/
codellama-13b-instruct.Q4_K_M.gguf

$ interpreter --model codellama-13b-instruct.Q4_K_M.gguf

Open Interpreter will use Code Llama for local execution. Use your arrow keys to set up the model.

[?] Parameter count (smaller is faster, larger is more capable): 7B
 > 7B
   13B
   34B
```

**Comment by kjyv** [comment]
@jordanbtucker Please reopen this again
